src/app/shared/pipes/age.pipe.spec.ts

```diff
@@ -1,8 +1,26 @@
 import { AgePipe } from './age.pipe';
+import { TestBed } from '@angular/core/testing';
 
 describe('AgePipe', () => {
+  let agePipe: AgePipe;
+
+  beforeEach(() => TestBed.configureTestingModule({
+    imports: [ ],
+    providers: [ AgePipe ]
+  }));
+
+  beforeEach(() => {
+    agePipe = TestBed.get(AgePipe);
+  });
+
   it('create an instance', () => {
     const pipe = new AgePipe();
     expect(pipe).toBeTruthy();
   });
+
+  it('should return 22', () => {
+    const age = agePipe.transform('1996-12-21T02:00:00.000Z');
+    console.log(age);
```

Comment:
You can add  ```"no-console": [true, "debug", "info", "time", "timeEnd", "trace", "log"]``` in your .tslint.json. 

This will emit an error, if you add a console.log in your project.